generators/contribute/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+const Generator = require('yeoman-generator')
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this override of "Generator". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2424)